### ISSUE TITLE
Towncrier generates links to issues without numbers

### ISSUE THREAD

**Issue by fleetingbytes** [twisted/towncrier#444]
I'm new to towncrier, so I just tried to follow the instructions in "How to keep a Changelog in Markdown". I created a random bugfix without an issue number:
```bash
towncrier create -c "A fix without an issue number!" +something-unique.fixed.md
```
but after `towncrier build --draft` it appears in the output with a link:

```md
### Bugfixes

- A fix without an issue number! [#+something-unique](https://github.com/twisted/my-project/issues/+something-unique)
```

I assume that this is not a towncrier bug, but a documentation bug and can be fixed with some additional logic in the suggested jinja template. Can we improve it? I have no previous experience with jinja, so I'm lost here.

**Comment by hynek** [comment]
You seem to have followed our latest docs. The problem is very simple: the + feature has not been part of a PyPI release yet.

**Comment by fleetingbytes** [comment]
ah, good, so I think I can just wait this one out. Thanks for the quick reply.

**Comment by pawelad** [comment]
I encountered the same issue - will this be 'fixed' by #456?

**Comment by adiroiban** [comment]
@pawelad the release candidate 22.12.0rc1 is available on PyPI, you can give it a try and see if the issue is fixed.

My guess is that it should be fixed.

Cheers
